core/src/main/scala/filodb.core/store/IngestionConfig.scala

```diff
@@ -86,7 +88,10 @@ object StoreConfig {
   /** Pass in the config inside the store {}  */
   def apply(storeConfig: Config): StoreConfig = {
     val config = storeConfig.withFallback(defaults)
-    StoreConfig(config.as[FiniteDuration]("flush-interval"),
+    val flushInterval = config.as[FiniteDuration]("flush-interval")
+    val fallbackMaxChunkTime = (flushInterval.toMillis * 1.1).toLong.millis
```

Comment:
1.1 is good, but let's leave a comment why we use that (extra margin of time to ensure chunks aren't ended prematurely?  etc etc)